app/src/main/java/org/mozilla/focus/fragment/HomeFragment.kt

```diff
@@ -107,7 +135,9 @@ class HomeFragment : Fragment() {
             addAll(BundledTilesManager.getInstance(context).getBundledHomeTilesList())
         }
 
-        adapter = HomeTileAdapter(uiLifecycleCancelJob, homeTiles, onUrlEnteredListener)
+        val hta = HomeTileAdapter(uiLifecycleCancelJob, homeTiles, onUrlEnteredListener)
+        adapter = hta
+        homeTileAdapter = hta
```

Comment:
lol what is this? why do you have so many adapters? This looks wrong. Do you need all these adapters? Since both of them are pointing to the same reference (hta) probably one of them is unnecessary. If you do need them both, add a comment as to why.

In general, this is bad code smell, because you have two similarly named variables, both being set to the same thnig. Before submitting something like this for review, do your own check to make sure you know why it's happening, and then add a comment explaining why it's necessary (because you *know* it's going to get flagged in review bc it looks so weird).